assignments/objects.js

```diff
@@ -49,16 +96,42 @@ let example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-let parent = {}
+let parent = {
+  name: 'Susan',
+  age: 70,
+  child: {
+    name: 'George',
+    age: 50,
+    grandchild: {
+      name: 'Sam',
+      age: 30,
+      speak: function() {
+        return `Hello my name is ${this.name}`
```

Comment:
So I'm sure you saw from our standup last night how it would perhaps be better to declare the speak function outside of the object and then assign it in each object as a method so that you don't have to write the same function multiple times. If you do this, remember not to call (`speak()`) the function when you set it on the key.

Reply:
Ya I was trying to do that but it wasn't working so I just did it the way I did to complete the project, i'll look into it and refactor